game/views/game_view.py

```diff
@@ -83,8 +90,9 @@ def load_map(self):
         self.exit_list = arcade.SpriteList()
 
         # Object Layers
-        levelfile = "game/assets/tilemaps/level1_.tmx"
-        self.object_layers = utils.process_objects(levelfile)
+        self.object_layers = utils.process_objects(
+            f"game/assets/levels/{self.level_list[self.level]}.tmx"
```

Comment:
should put this in a variable, you have the exact same fstring above

Reply:
😨 Where

Reply:
oh
yike
sure